providers/kit/examples_test.go

```diff
@@ -30,11 +30,9 @@ func Example_initializationWithCustomLevels() {
 	// Create a server, make sure we put the tags context before everything else.
 	_ = grpc.NewServer(
 		grpc.ChainUnaryInterceptor(
-			tags.UnaryServerInterceptor(),
 			logging.UnaryServerInterceptor(kit.InterceptorLogger(logger), opts...),
 		),
 		grpc.ChainStreamInterceptor(
```

Comment:
No more tags interceptor for inserting key-values in the context, that is a superb cleanup of code!